docs/templates/auth.md.j2

```diff
@@ -0,0 +1,309 @@
+* **access_token** is the bearer token you need to include on API calls. See [Using an Access Token on API Calls](#using-an-access-token-on-api-calls).
+* **expires_in** is the number of seconds for which the access token is valid, from the time the token is issued.
+* **refresh_token** is the token you would use on a refresh request. See [Refreshing an Access Token](#refreshing-an-access-token).
+* **refresh_expires**_in is the number of seconds for which the refresh token is valid. This is always longer than the access token validity period.
+
+
+## Requesting a Custom Access Token
+
+You can use the password-granted access token. However, you can also request a custom access token. With a custom token, you can supply a subject name to help differentiate token usage (for your own purposes). You can also request specific validity periods if the default values returned for password tokens do not fit your requirements.
+
+### Before You Begin
+
+You must first get a password-granted access token before getting a custom token. See [Requesting a Password-Granted Access Token](#requesting-a-password-granted-access-token).
+
+### Procedure
+
+#### Step 1
+
+Create the JSON object for the custom access token grant.
+
+```json
+{
+  "grant_type": "custom_token",
+  "access_token": "string",
+  "desired_expires_in": 0,
+  "desired_refresh_expires_in": 0,
+  "desired_subject": "string",
+  "desired_refresh_count": 0
+}
+```
+Where:
+
+* **access_token** is a valid password-granted access token.
+* **desired_expires_in** is an integer representing the number of seconds for which the custom access token will be valid. In comparison, the password-granted tokens are valid for 1800 seconds.
+* **desired_refresh_expires_in** is an integer representing the number of seconds for which the custom refresh token will be valid. If you obtain a refresh token, ensure that this value is larger than the **desired_expires_in** value. In comparison, the password-granted refresh tokens are valid for 2400 seconds. This parameter is not required if you specify 0 for **desired_refresh_count**.
+* **desired_subject** is a name you give to the custom token.
+* **desired_refresh_count** is the number of times you want to be able to refresh the token. Specify 0 if you do not want to get a refresh token. When you do not have a refresh token, you must obtain a new access token when the existing one expires.
+
+For example, the following requests a custom token for api-client that expires in 2400 seconds, with a refresh token that expires in 3000 seconds. The token can be refreshed 3 times.
+```json
+{
+  "grant_type": "custom_token",
+  "access_token": "eyJhbGciOiJIUzI1NiJ9.eyJpYXQiOjE1MDI4MzI2NjcsInN1YiI6ImFkbWluIiwianRpIjoiMGM3ZDBmNDgtODIwMS0xMWU3LWE4MWMtMDcwZmYzOWU3ZjQ0IiwibmJmIjoxNTAyODMyNjY3LCJleHAiOjE1MDI4MzQ0NjcsInJlZnJlc2hUb2tlbkV4cGlyZXNBdCI6MTUwMjgzNTA2NzQxOSwidG9rZW5UeXBlIjoiSldUX0FjY2VzcyIsIm9yaWdpbiI6InBhc3N3b3JkIn0.b2hI6fVA_GbmhCOPM-ZUx6IC8SgCk1AkHXI-llV0r7s",
```

Comment:
Just to double-check, but these tokens were taken from the Support Center and fine to published as is, right?

Reply:
good question. I checked that source page was public - as result, I assume it's not sensitive.